src/components/SpellingVariantInput.vue

```diff
@@ -11,16 +12,20 @@ interface Props {
 interface WikitMenuItem {
 	label: string;
 	description: string;
+	value: string;
 }
 
 const props = defineProps<Props>();
 
 const languageCodesProvider = useLanguageCodesProvider();
 
-const wbLexemeTermLanguages = languageCodesProvider.getLanguageCodes().map( ( lang ) => ( {
-	label: lang,
-	description: '',
-} ) );
+const wbLexemeTermLanguages = languageCodesProvider.getLanguageCodes().map(
+	( [ code, name ]: [ string, string ] ) => ( {
+		label: `${name} (${code})`,
```

Comment:
Hm, should this be localized? ({{optional}}, identical to e.g. authprovider-confirmlink-option)

Reply:
Done, though this had somewhat of a ripple effect because we're matching on that label now have to test based on how the messages looks like and how it is replaced. I created #143 for this